examples/default.yaml

```diff
@@ -353,6 +353,14 @@ hostResolver:
 # - 1.1.1.1
 # - 1.0.0.1
 
+# Lima volumes to attach to the instance. The volumes will be accessible from inside
+# the instance, labeled by name. (e.g. if the volume is named "data", it will be labeled
+# "lima-data" inside the instance).
```

Comment:
Where is the mount point of the volume?

Reply:
Will add in the next commit. Planning on mounting them under `/mnt/lima-${VOLUME_NAME}`

Reply:
The mount point should be documented right here in `default.yaml`.